comment/templates/comment/block/block_modal.html

```diff
@@ -0,0 +1,39 @@
+{% load i18n %}
+{% load comment_tags %}
+<div id="blockModal" class="comment-modal">
+    <div class="comment-modal-dialog" role="document">
+        <div class="comment-modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">{% block title %}{% trans "Are you sure you want to block this user?" %}{% endblock title %}</h5>
+                <button class="modal-close-btn" aria-label="{% trans "Close" %}">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body">
+                <form class="js-comment-block-form" data-url="{% url 'comment:toggle-blocking' %}">
+                {% block reason_input %}
+                    <div class="row">
+                        <div class="col-3">
+                            <label for="reason">Reason:</label>
```

Comment:
```suggestion
                            <label for="reason" class="comment-translatable"> {% trans "Reason" %} >
```
The class "comment-translatable" will allow tests to check if the content inside the element is available for translation or not, provided you are using the function `assertHttmlTranslated` when rendering this.
https://github.com/Radi85/Comment/blob/bdfdea180680b0f83730e4fdf350eed5411b539e/comment/tests/base.py#L199-L205
https://github.com/Radi85/Comment/blob/bdfdea180680b0f83730e4fdf350eed5411b539e/comment/tests/base.py#L29-L32
OR
you may also add this `label` attribute here for future purposes.
https://github.com/Radi85/Comment/blob/bdfdea180680b0f83730e4fdf350eed5411b539e/comment/tests/base.py#L26

Reply:
hmmm, this will always require us to update test translation files.